exercises/practice/grep/grep.wren

```diff
@@ -0,0 +1,11 @@
+import "io" for File
+
+class Grep {
+  static grep(pattern, files) {
+    Fiber.abort("Remove this statement and implement this function")
+  }
+
+  static grep(flags, pattern, files) {
```

Comment:
Is this the JS API?  This is so terrible, ugh... I don't support you'd want to rework it?  Ok to say no.

Feels like flags should be LAST param, not first.

Reply:
No problem.